### PR TITLE
CME & SUclass Theme Stage Testing Changes

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd & @kluo 

I'll add all the commits here

### CODE REVIEW

**Comment by caesar2164** [comment]
[Fix Register Button for CME](https://trello.com/c/zN53fcnS/199-fix-register-button-for-cme)

**Comment by caesar2164** [comment]
[Fix Register Button for CME](https://trello.com/c/zN53fcnS/199-fix-register-button-for-cme)

**Comment by caesar2164** [comment]
[Confirm that SYSADMIN link is present on suclass](https://trello.com/c/tVsm0zWr/202-confirm-that-sysadmin-link-is-present-on-suclass)

File: cme/lms/templates/header.html [hunk 1]
```diff
@@ -12,18 +12,21 @@
...
+    <%block name="navigation_global_links_authenticated">
+        %if settings.FEATURES.get('ENABLE_SYSADMIN_DASHBOARD','') and user.is_staff:
+            <li class="item">
+                ## Translators: This is short for "System administration".
+                <a href="${reverse('sysadmin')}">${_("Sysadmin")}</a>
+            </li>
+        %endif
+    </%block>
+% else:
+    <%block name="navigation_global_links">
```
Comment (stvstnfrd) [inline cme/lms/templates/header.html]: Have you taken a look at `./lms/templates/navigation.html` in the platform [1]?

It looks like the register link is originally set in `navigation_other_global_links`, not `navigation_global_links`.

I think the default logic actually gives us what we want.
Assuming so, let's remove this override and drop the one below too.

[1] https://github.com/Stanford-Online/edx-platform/blob/master/lms/templates/navigation.html#L154-L171

File: cme/lms/templates/header.html [hunk 2]
```diff
@@ -12,18 +12,21 @@
...
+        <li class="item nav-global-04">
+            <a class="btn-neutral" href="/register${login_query()}">${_("Register")}</a>
+        </li>
+    </%block>
 %endif
+
+<%block name="navigation_other_global_links"></%block>
```
Comment (stvstnfrd) [inline cme/lms/templates/header.html]: See note above about removing.

File: cme/lms/templates/header.html [hunk 3]
```diff
@@ -12,18 +12,21 @@
...
-                    <a href="${reverse('sysadmin')}">${_("Sysadmin")}</a>
-                </li>
-            %endif
-        </%block>
-    %endif
-%else:
-    <%block name="navigation_global_links"></%block>
-    <%block name="navigation_other_global_links"></%block>
+% if UserProfile.has_registered(user):
+    <%block name="navigation_global_links_authenticated">
```
Comment (stvstnfrd) [inline cme/lms/templates/header.html]: Similarly, I think the default value suffices here.

Let's try unforking.

File: suclass/lms/templates/header.html [hunk 4]
```diff
@@ -12,18 +12,16 @@
...
-                    ## Translators: This is short for "System administration".
-                    <a href="${reverse('sysadmin')}">${_("Sysadmin")}</a>
-                </li>
-            %endif
-        </%block>
-    %endif
-%else:
-    <%block name="navigation_global_links"></%block>
-    <%block name="navigation_other_global_links"></%block>
+% if UserProfile.has_registered(user):
```
Comment (stvstnfrd) [inline suclass/lms/templates/header.html]: Let's try unforking all of these blocks too.
Most links are already conditional based on feature flags.

**Review by stvstnfrd — DISMISSED** [review]

File: suclass/lms/templates/header.html [hunk 5]
```diff
@@ -12,18 +12,16 @@
   <img src="${static.url('images/stanford-S-logo.png')}" height="35" alt="Home Page" />
```
Comment (stvstnfrd) [inline suclass/lms/templates/header.html]: Thanks to comp theming, we may be able to unfork this block.

The template tries to pull the logo from a number of locations first,
but falls back to `images/logo.png`.
I think if we renamed the file to the above, it should be found automatically.

**Comment by stvstnfrd** [comment]
Hrmm, if we can unfork each of these mentioned blocks, we can actually remove this file entirely :)

File: cme/lms/templates/header.html (hunk 3 above)
Comment (caesar2164) [reply cme/lms/templates/header.html]: We can't cause platform `About` isn't configurable through settings.

Comment (stvstnfrd) [reply cme/lms/templates/header.html]: Let's just wrap it in a feature flag and make it configurable then :)

**Comment by stvstnfrd** [comment]
+1 after you test locallly

**Pull request merged**